### DISCUSSION TITLE
Simulate honey stickiness to the solid

### DISCUSSION THREAD

**Discussion by NeonBohdan** [InteractiveComputerGraphics/SPlisHSPlasH#209] [Q&A] [answered]
@janbender thanks for the great project again)
I'm trying to simulate the winding of honey on a rotating cylinder

But as I can see there is no actual **surface adhesion**, only `viscosityBoundary`
Can it be somehow added? For example a similar effect is achieved through surface tension between two drops
Or is it only possible by adding `AnimationFields`

My config [Honey.json](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/files/9896479/Honey.json.txt)

___
And some small question

For some reason emitter generates some constant amound of particles, ignoring `emitEndTime`
I can't figure out why

**Comment by Theodotus1243** [comment]
Sorry, used wrong account

**Comment by Theodotus1243** [comment]
Reporting, that I've solved **surface adhesion** problem by adding `AnimationFields`
You can see it config [Honey.json](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/files/9913690/Honey.json.txt)
___
But still there is a problem, that if I remove `emitEndTime` particles will not emit infinitely
@janbender do You know why it can happen

**Comment by janbender** [comment, marked as answer]
There is a max number of particles that are emitted. This can be also set in the scene file.
